ang/civicase/ContactCaseTab.html

```diff
@@ -0,0 +1,38 @@
+
+<div id="bootstrap-theme" class="clearfix civicase__container">
+  <h1 crm-page-title class="element-invisible">Related cases to the user</h1>  <!-- Added for SEO purpose (This will be hidden through css) -->
+  <div class="civicase__contact-cases clearfix">
```

Comment:
Should be `civicase__contact-cases-tab`. Also please update other classes which start with `civicase__contact-cases`.